stacks/java-wildfly/devfile.yaml

```diff
@@ -1,16 +1,16 @@
 schemaVersion: 2.1.0
 metadata:
   name: java-wildfly
-  version: 1.1.0
-  website: https://wildfly.org
   displayName: WildFly Java
-  description: Upstream WildFly
+  description: Java application using WildFly 26.1.0
```

Comment:
add jvm version

Reply:
I was able to find the JVM version for the Open/WebSphere Liberty devfiles, but this is the only one which I cannot find this info :)

Reply:
@elsony a reminder for this one. I was able to fix the rest of the descriptions but I couldn't find the JVM version for the wildly :)

Reply:
It could still run on 8 but 11 is the best choice, all the more so as the next versions will be 11 or more .